### DISCUSSION TITLE
Will private git hosted on on-prem or enterprise git hosted onprem expose Graphql APIs ?

### DISCUSSION THREAD

**Discussion by manishjain5238** [community/community#59672] [Enterprise]
### Select Topic Area

Question

### Body

whether the private Git infrastructure, regardless of whether it is hosted on-premises or on GitHub Enterprise on-premises, provides Graphql APIs ?

**Comment by byrneh** [comment]
GitHub Enterprise Server (self hosted on-premie or in Cloud) porvides GrappQL api.
Refer ro the GiThub Documetaion specific to te version you have installed
[https://docs.github.com/en/enterprise-server@3.9/graphql](https://docs.github.com/en/enterprise-server@3.9/graphql)
